docs/framework/migration-guide/versions-and-dependencies.md

```diff
@@ -77,5 +77,5 @@ Each version of the .NET Framework contains the common language runtime (CLR), t
 ## See also
 
 - [Version Compatibility](version-compatibility.md)
-- [Microsoft .NET Framework Support Lifecycle Policy](https://go.microsoft.com/fwlink/?LinkId=196607)
+- [Microsoft .NET Framework Support Lifecycle Policy](https://support.microsoft.com/help/17455/lifecycle-faq-net-framework)
```

Comment:
This is a good change to do it broadly with a search and replace to see if there are more after we merge this PR 

```suggestion
- [.NET Framework Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-framework)
```